.github/CODEOWNERS

```diff
@@ -43,9 +43,6 @@
 # PRLabel: %Batch
 /sdk/batch/ @dpwatrous @paterasMSFT @zfengms @deyaaeldeen
 
-# PRLabel: %Communication
```

Comment:
Please leave the PRLabel  comment (L46) and the path L47 in place and just remove the associated handles.

Reply:
Done